src/pages/Home.tsx

```diff
@@ -47,8 +49,39 @@ export const Home = () => {
         gap="md"
         direction={isMobile ? 'column' : 'row'}
       >
+        {children}
+      </Flex>
+    </Box>
+  );
+
+  let content = <></>;
+
+  if (isLoading) {
+    content = (
+      <HomeWrapper>
+        <Box w={isMobile ? '100%' : '50%'} maw={500}>
+          <SectionText mb="md">Live Rounds</SectionText>
+          <Stack>
+            <FeedSkeletonCard />
+            <FeedSkeletonCard />
+            <FeedSkeletonCard />
+          </Stack>
+        </Box>
+        <Box w={isMobile ? '100%' : '50%'} maw={500}>
+          <SectionText mb="md">Past Votes</SectionText>
+          <Stack>
+            <FeedSkeletonCard />
+            <FeedSkeletonCard />
+            <FeedSkeletonCard />
+          </Stack>
+        </Box>
+      </HomeWrapper>
+    );
+  } else {
+    content = (
```

Comment:
We'll need to fix this pattern. 

Direct variable assignment inside React Components has a lot of major issues. Mostly to do with state updates and the component render cycle. I can show you how to work around this.